src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -2298,13 +2298,15 @@ namespace nupic {
   {
     SpatialPooler sp;
     Real synPermConnected = 0.2;
-    Real synPermActiveInc = 0.05;
+    Real synPermMax = 1.0;
+
     sp.setSynPermConnected(synPermConnected);
-    sp.setSynPermActiveInc(synPermActiveInc);
+    sp.setSynPermMax(synPermMax);
+
     for (UInt i = 0; i < 100; i++) {
       Real permVal = sp.initPermConnected_();
       NTA_CHECK(permVal >= synPermConnected &&
-                permVal <= synPermConnected + synPermActiveInc / 4.0);
+                permVal <= synPermMax);
```

Comment:
Not a big deal, but you don't need to explicitly set it to 1.0. You can instead just use `permVal <= sp.getSynPermMax()' in the if statement. Saves a few lines :-)